Mechanical/Body Assembly/README.md

```diff
@@ -19,22 +20,191 @@ The body is the housing of all the electronics for the rover. It is the attachme
+
+First we need to drill a hole in one of the 9x12 Aluminum plates **S35** because we will need a hole of just over 0.5 in diameter for the differential pivot mount. There is already a small hole drilled in the location we want to use, but it needs to be widened substantially. Start with the drill #23 and drill the hole shown by Figure 1. Repeat this with drill sizes stepping up until you get to a drill of 0.5 in.
+
+Take the 0.5 in hollow rod **S19** and make sure it spins freely in the hole you have created. If it does not, drill the hole slightly larger or sand/file the hole until the rod spins with no resistance (The 0.5 in hollow rod must spin _freely_ while mounted inside the bearing blocks. See step 2.2 Differential pivot for example. It may help to follow step 2.2 in this document to test if you have enough clearance)
+
+| <img src="Pictures/Fabrication/9x12_Plate_cut.png" width="50%"> <img src="Pictures/Fabrication/9x12_Plate_cut2.png" width="40%"> |
+|:-:|
+| Figure 1: Drilling the Aluminum Plate |
+
+### 7.2. Laser Cut Parts
+
+The front and back panel of the rover are designed to be made from laser cut acrylic, as to keep them modular to design different things to mount there as wanted. The 2D cutout files  are the .DXF files  and can be found on the github under Mechanical - Body Assembly - Laser Cut Parts
+
+If you do not have access to a lases cutter there is an online service which you can order these from below: https://www.sculpteo.com
+
+To get the above parts from Sculpteo, go to Laser cutting and then upload these files (with mm selected as units). Hit Next. Make sure scale is set to 100%, change the material to Acrylic, have thickness to 1/8 inch, and then select whatever color you wish.
+
+## 8. Mechanical/Structural Assembly
+
+**DISCLAIMER** All images were created using CAD that has #6-32 Socket head cap screws, but the parts list calls for button head screws. The button head screws are correct.
+
+### 8.1. Chassis
+
+| Item | Ref | Qty | Image |
+| :--- | :-- | :-- | :---: |
+| Dual Side mount A            | S17 | 4 | <img src="../../images/components/Structural/S17.jpg" width="25%"> |
+| 4.5"x12" Aluminum Plate      | S37 | 2 | <img src="../../images/components/Structural/S37.jpg" width="40%"> |
+| 1" PVC Clamp                 | S24 | 1 | <img src="../../images/components/Structural/S24.jpg" width="70%"> |
+| 9"x12" Aluminum Plate        | S35A | 1 | <img src="../../images/components/Structural/S35.jpg" width="40%"> |
+| #6-32x1/4" Button Head Screw | B1 | 16 | <img src="../../images/components/Screws/B1.png" width="15%"> |
+| #6-32x3/8" Button Head Screw | B2 | 4 | <img src="../../images/components/Screws/B2.png" width="15%"> |
+| Allen Key Set                | | | <img src="../../images/components/Tools/D2.jpeg" width="30%"> |
+| 5/16" Wrench                 | | | <img src="../../images/components/Tools/D1.jpg" width="30%">  |
```

Comment:
Can you split this into two columns to condense it?

Reply:
Yeah, we'll split into two columns. It is pretty big as is.